src/diagram_text_emitter.rs

```diff
@@ -57,6 +59,11 @@ fn get_ticket_numbers_from_blocker_description(blocker_desc: &str) -> Vec<String
         .map(|cap| cap.get(1).map_or("".to_owned(), |m| m.as_str().to_owned()))
         .collect::<Vec<String>>();
     tickets.append(&mut full_url_ticket_ids);
+    let mut alternate_full_url_ticket_ids = ALTERNATE_URL_REGEX
+        .captures_iter(blocker_desc)
+        .map(|cap| cap.get(2).map_or("".to_owned(), |m| m.as_str().to_owned()))
```

Comment:
Ah the addition of the second capture group in the expression took a second, but this makes sense.